test/system/roles/analyst/analyst_test.rb

```diff
@@ -25,10 +23,11 @@ class AnalystTest < ApplicationSystemTestCase
   end
 
   def view_analytics(user_label)
-    jurisdiction_id = login_and_view_analytics(user_label)
-    analytic_id = Analytic.where(jurisdiction_id: jurisdiction_id).order(created_at: :desc).first['id']
-    @@analyst_dashboard_monitoree_counts_verifier.verify_monitoree_counts(analytic_id)
-    @@analyst_dashboard_monitoree_snapshots_verifier.verify_monitoree_snapshots(analytic_id)
+    login_and_view_analytics(user_label)
+    # Commenting these out. The logic here needs to be re-worked here.
+    # Many of the Analytics tests need to be overhauled
+    # analytic_id = Analytic.where(jurisdiction_id: jurisdiction_id).order(created_at: :desc).first['id']
```

Comment:
@holmesie is it okay to push this off to a future task?

Reply:
I would prefer these tests to be fixed/updated rather than deleted

Reply:
Agree with @timwongj, fixing/updating these is definitely something that falls under this task.

Reply:
I agree, but that may take some time, and Kelly said she wanted this to come out ASAP, which is why I removed them (to save time). What should I do?

Reply:
I would probably let Kelly know that it would take more time, don't want to rush in untested code.

Reply:
@timwongj I spoke with Adam and apparently this code can be deployed on UAT (which was Kelly wanted) without being merged in. So problem solved. Will continue to flesh out the tests on this PR